neqo-transport/tests/sim/taildrop.rs

```diff
@@ -71,6 +75,9 @@ impl TailDrop {
             dropped: 0,
             delivered: 0,
             maxq: 0,
+            start_time: None,
+            last_sent: None,
+            sent_bytes: 0,
```

Comment:
Given the number of statistical items we now have, it might pay to factor them out into a TaildropStats struct.  Then the core functioning of this doesn't get swamped with all the stats.